docs/topics/forms/formsets.txt

```diff
@@ -432,16 +508,18 @@ If ``validate_max=True`` is passed to
 that the number of forms in the data set, minus those marked for
 deletion, is less than or equal to ``max_num``.
 
+.. code-block:: pycon
```

Comment:
This change here (and the code-block below) look like related cleanups. 

I think you could open a PR with just these changes now and we could get that merged in right away (and back ported).

Reply:
Hi @smithdc1, thanks for the suggestion! I appreciate your feedback. For now, I'll focus on removing the unrelated cleanups from this PR to keep it focused on its current goal. Later, I'll open a new PR for them.